### DISCUSSION TITLE
Changing Table's bg color

### DISCUSSION THREAD

**Discussion by goesbyabhi** [saadeghi/daisyui#1269] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83471520/197327219-178d8057-f6c3-4b91-8591-b50c69b5916e.png)
I am trying to change this table's background color. I tried adding `bg-primary-content` to change the bg color to white. But it is not working for some reason. I have tried removing the `table-zebra` class and tried to change the bg color but it is still not working.  Am I doing it wrong?

**Comment by saadeghi** [comment, marked as answer]
The whole table doesn't have a background color. Every cell has.  
So the easiest way would be a class name like `[&_*]:bg-primary` on `<table>`.  
Example: https://play.tailwindcss.com/MCewKL2h7O

**Reply by goesbyabhi** [reply]
Thank you so much! This cleared my problem and helped me!!
